CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.5.0] - 2021-11-15
```

Comment:
I think this may be a large enough change to bump to `v2.0.0`.   Could you add a note about adding functionality for our DR environment?